adhocracy4/exports/views.py

```diff
@@ -84,14 +88,27 @@ def get_field_data(self, item, name):
         except TypeError:
             pass
 
+        # Return the get_field_display value for choice fields
+        get_field_display_method = getattr(
+            item, 'get_{}_display'.format(name), None)
+        if get_field_display_method and callable(get_field_display_method):
+            return get_field_display_method()
+
+        fieldtype = self._get_field_type(item, name)
+
+        if fieldtype and fieldtype == 'RichTextField':
+            text = getattr(item, name, '')
+            return self.get_rich_text_field_data(text)
+
         # Finally try to get the fields data as a property
         return str(getattr(item, name, ''))
 
     def get_link_data(self, item):
         return self.request.build_absolute_uri(item.get_absolute_url())
 
-    def get_description_data(self, item):
-        return strip_tags(item.description).strip()
+    def get_rich_text_field_data(self, text):
```

Comment:
if we keep this general method i would prefer to rename it, as the naming scheme was `get_X_data` where `X` is standing for a (virtual) field which is represented as a column in the export.

Reply:
Sure, what would you suggest? We could extend this for more fields